meilisearch-http/src/routes/search.rs

```diff
@@ -94,144 +98,19 @@ pub async fn search_with_url_query(ctx: Request<Data>) -> SResult<Response> {
         search_builder.attributes_to_highlight(attributes_to_highlight);
     }
 
-    if let Some(filters) = query.filters {
-        search_builder.filters(filters);
+    if let Some(filters) = &params.filters {
+        search_builder.filters(filters.to_string());
     }
 
-    if let Some(timeout_ms) = query.timeout_ms {
+    if let Some(timeout_ms) = params.timeout_ms {
         search_builder.timeout(Duration::from_millis(timeout_ms));
     }
```

Comment:
This parameter doesn't exists anymore [even in the doc](https://docs.meilisearch.com/references/search.html#search-in-an-index). Could you remove it, please?